crates/tabby-common/src/registry.rs

```diff
@@ -12,7 +12,10 @@ pub struct ModelInfo {
     pub prompt_template: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub chat_template: Option<String>,
+    #[serde(skip_serializing_if = "Vec::is_empty")]
     pub urls: Vec<String>,
+    #[serde(skip_serializing_if = "Vec::is_empty")]
+    pub segmented_urls: Vec<String>,
```

Comment:
```suggestion
    pub partition_urls: Vec<Vec<String>>,
```

Allow partitioned urls can also have their mirrors.